recipes/extra-cmake-modules/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class ExtracmakemodulesConan(ConanFile):
+    name = "extra-cmake-modules"
+    license = ("MIT", "BSD-2-clause", "BSD-3-clause")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://api.kde.org/ecm/"
+    topics = ("conan", "cmake")
+    description = "KDE's CMake modules"
+    generators = "cmake"
+    no_copy_source = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions['BUILD_HTML_DOCS'] = False
+        cmake.definitions['BUILD_QTHELP_DOCS'] = False
+        cmake.definitions['BUILD_MAN_DOCS'] = False
+        cmake.definitions['SHARE_INSTALL_DIR'] = os.path.join(self.package_folder, "res")
```

Comment:
Are the hooks fine with this folder?

Reply:
actually I had to use this folder because of the hook.
```
The binary packages shouldn't contain any other files or folder except the following: ["lib", "bin", "include", "res", "licenses"]
```
https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#kb-h013-default-package-layout

Reply:
Thanks for the link!